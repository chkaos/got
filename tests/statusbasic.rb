#!/usr/bin/env ruby

require 'tmpdir'

expected = IO.read(File.join(File.dirname(__FILE__), "statusbasic.out"))

dir = Dir.mktmpdir
Dir.chdir(dir)

`got init`

File.write('file1.txt', 'First file')
File.write('file2.txt', 'Second file')

`got commit -m "Test Message" -a "Test Author"`

File.write('file3.txt', 'Third file')
File.write('file2.txt', 'Modified second file')
File.delete('file1.txt')

result = `got status`

if result != expected
  exit 1
end
