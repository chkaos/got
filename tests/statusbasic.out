Comparing working directory to empty tree.
[92;40mA file1.txt[0m
[92;40mA file2.txt[0m
